Intermittent test failures in new multi dimensional functions

```
======================================================================
FAIL: test_function_measure_multi_dimensional_shifts_3d (test.MultiDimensionalProcessing_test.TestMultiDimensionalProcessing) [Test for a 2D collection of 1D data, measure shift of data dimensions along collection axis]
----------------------------------------------------------------------
Traceback (most recent call last):
  File "D:\a\niondata\niondata\nion\data\test\MultiDimensionalProcessing_test.py", line 1[8](https://github.com/nion-software/niondata/actions/runs/3049104430/jobs/4914843216#step:8:9)5, in test_function_measure_multi_dimensional_shifts_3d
    self.assertTrue(numpy.allclose(result.data, -1.0 * (shifts - shifts[numpy.unravel_index(reference_index, shifts.shape)]), atol=0.5))
AssertionError: False is not true

----------------------------------------------------------------------
Ran 106 tests in 1.34[9](https://github.com/nion-software/niondata/actions/runs/3049104430/jobs/4914843216#step:8:10)s
```

--- a/nion/data/test/MultiDimensionalProcessing_test.py
+++ b/nion/data/test/MultiDimensionalProcessing_test.py
@@ -13,6 +13,13 @@ _ = gettext.gettext
 
 class TestMultiDimensionalProcessing(unittest.TestCase):
 
+    def setUp(self) -> None:
+        self.__random_state = numpy.random.get_state()
+        numpy.random.seed(42)
+
+    def tearDown(self) -> None:
+        numpy.random.set_state(self.__random_state)
+
     def test_function_apply_multi_dimensional_shifts_4d(self) -> None:
         with self.subTest("Test for a sequence of SIs, shift collection dimensions along sequence axis"):
             shape = (5, 2, 3, 4)

